scipy/stats/_distribution_infrastructure.py

```diff
@@ -3481,9 +3481,7 @@ def make_distribution(dist):
     .. note::
 
         `make_distribution` does not work with all instances of `rv_continuous`.
-        Known failures include 'genpareto', 'genextreme', 'genhalflogistic',
-        'irwinhall', 'kstwo', 'kappa4', 'levy_stable', 'norminvgauss',
-        'tukeylambda', and `vonmises`.
+        Known failures include 'levy_stable' and `vonmises`.
```

Comment:
backticks

![image](https://github.com/user-attachments/assets/35b7d78e-aeff-4b59-84eb-43340f148597)

Reply:
Oops. Must have copy-posted from a list that had quotes.